Dockerfile

```diff
@@ -1,5 +1,16 @@
-FROM ubuntu:16.04
+FROM golang:latest as builder
```

Comment:
I am not a fan of using `latest` in this case, which can accidentally break the build process -- can we statically set this for now?